third_party/amd/backend/compiler.py

```diff
@@ -30,12 +31,25 @@ class HIPOptions:
     matrix_inst_shape: int = 0
     max_num_imprecise_acc_default: int = 0
 
+    @staticmethod
+    def get_warp_size(arch: str) -> int:
+        # 64 is not supported for RDNA for now
+        if 'gfx10' in arch or 'gfx11' in arch:
```

Comment:
could this just be checking for `gfx1`?

Reply:
I agreed with @zhanglx13 in AMD fork
https://github.com/ROCm/triton/pull/491#discussion_r1472227631